### ISSUE TITLE
St. Petersburg, FL 

### ISSUE THREAD

**Issue by ritazwiee** [defund12/defund12.org#639]
To: mayor@stpete.org, cynthia.sheppard@stpete.org, council@stpete.org, robert.blackmon@stpete.org, brandi.gabbard@stpete.org, ed.montanari@stpete.org, darden.rice@stpete.org, gina.driscoll@stpete.org, deborah.figgs-sanders@stpete.org, lisa.wheeler-bowman@stpete.org, amy.foster@stpete.org

Dear St. Petersburg City Coucil, 

My name is [NAME] and I am a citizen of St. Petersburg, FL. I am writing to you in regards to the national outcry over the murder of George Floyd, and concern over the conduct of the St. Petersburg Police Department.This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. Our city has been at the forefront of much of this action. Accordingly, it has come to my attention that the budget for 2021 is being decided as these protests continue.

I demand that the City Council begin meaningfully defunding the St. Petersburg Police Department and re-allocate those funds to programs proven to more effectively promote a safe and equitable community: community-based mental health services, substance abuse treatment services, affordable housing programs, and more. I demand a budget that reflects the actual needs of St. Petersburg residents.

History has shown that police "reform" is not enough. We must take a hard look at the ways that the current system in place fails to serve--and in fact actively harms--our community, and come together to reimagine the role of police in our city. We can create a new city, safe and beautiful for all people of St. Petersburg who call this place their home. Through listening and learning from the communicated demands, needs, and desires of the Black community, we can build a sustainable and safe city for today and tomorrow.

Thank you for your time,
[YOUR NAME]
[YOUR ADDRESS]
[YOUR EMAIL]
[YOUR PHONE NUMBER]

**Comment by dionlarson** [comment]
Thanks @ritazwiee. I was just about to start putting this together myself -- you saved me time. I will submit a pull request so it can get added to the website.

**Comment by milesalex** [comment]
@dionlarson Please review the [teams](TEAMS.md) before submitting PR

**Comment by dionlarson** [comment]
## Changelog

1. Coucil -> Council
2. citizen -> resident
3. added missing space
4. sentence structure of last paragraph

## Questions

Is there too much "we" at the end?

## Current

To: mayor@stpete.org, cynthia.sheppard@stpete.org, council@stpete.org, robert.blackmon@stpete.org, brandi.gabbard@stpete.org, ed.montanari@stpete.org, darden.rice@stpete.org, gina.driscoll@stpete.org, deborah.figgs-sanders@stpete.org, lisa.wheeler-bowman@stpete.org, amy.foster@stpete.org

Dear St. Petersburg City Council,

My name is [NAME] and I am a resident of St. Petersburg, FL. I am writing to you in regards to the national outcry over the murder of George Floyd, and concern over the conduct of the St. Petersburg Police Department. This past week, our nation has been gripped by protests calling for a rapid and meaningful reconsideration of the role of policing in communities as well as an end to racism and anti-Blackness in America. Our city has been at the forefront of much of this action. Accordingly, it has come to my attention that the budget for 2021 is being decided as these protests continue.

I demand that the City Council begin meaningfully defunding the St. Petersburg Police Department and re-allocate those funds to programs proven to more effectively promote a safe and equitable community: community-based mental health services, substance abuse treatment services, affordable housing programs, and more. I demand a budget that reflects the actual needs of St. Petersburg residents.

History has shown that police "reform" is not enough. We must take a hard look at the ways that the current system in place fails to serve, and in fact actively harms, our community. We must come together to reimagine the role of police in our city. We can create a new city, safe and beautiful for all people of St. Petersburg who call this place their home. Through listening and learning from the communicated demands, needs, and desires of the Black community, we can build a sustainable and safe city for today and tomorrow.

Thank you for your time,
[YOUR NAME]
[YOUR ADDRESS]
[YOUR EMAIL]
[YOUR PHONE NUMBER]

**Comment by ritazwiee** [comment]
Possible edit: 
"History has shown that police "reform" is not enough. We must take a hard look at the ways that the current system in place fails to serve, and in fact actively harms, our community. We must come together to reimagine the role of police in our city. I believe that a new city, safe and beautiful, can be created for all people of St. Petersburg who call this place their home. Through listening and learning from the communicated demands, needs, and desires of the Black community, there can be a sustainable and safe city for today and tomorrow."

**Comment by dionlarson** [comment]
Thank you for your help @ritazwiee. I've added your suggestions to my above message.

It looks like the review process is on pause until after a meeting tonight at 9pm to better coordinate the full review process. I'll pick this up again once approved!

**Comment by dionlarson** [comment]
Proposed [2020-2021 budget](https://www.stpete.org/city_departments/docs/FY21%20Operating%20Budget%20Workshop%20Workbook.pdf) that is submitted for approval on July 10th 2020. I have added some on the following in between paragraphs one and two.

- $116,368,000 to police, 38.85% of $299,509,850 total budget
- "In the FY20 Adopted Budget, we started a three-year plan to increase the department’s sworn personnel from 562 to 600 by including funding for 13 full-time Police Officers. For the second year of the plan, the city has applied for a COPS grant to receive partial funding for the remaining 25 full-time Police Officers needed to complete the plan. "
